src/components/screens/AvailableVehicles.tsx

```diff
@@ -65,8 +79,9 @@ export const AvailableVehicles = ({
           <Fragment key={item.id}>
             <ListItem style={styles.container}>
               {!!iconName && <CategoryIcon />}
-              <ListItem.Content>
+              <ListItem.Content style={styles.contentContainer}>
                 <RegularText>{item.name}</RegularText>
+                <RegularText>{item.count}</RegularText>
```

Comment:
This could be realized with a `Badge` like:

```
</ListItem.Content>

<Badge value={count} badgeStyle={styles.badge} textStyle={styles.badgeText} />
```

I have implemented a badge already in hb/meinquartier.

To avoid having to set a style for the content.
On the other hand, the badge itself needs styling. But this could move in an own badge component to be reusable.

Reply:
It hasn't been done yet, but it could be possible. 👍

Reply:
all right, i have linked it in https://github.com/smart-village-solutions/smart-village-app-app/pull/969/files#r1695535521